recipes/srfi

```diff
@@ -0,0 +1,2 @@
+(srfi :fetcher github :repo "srfi-explorations/emacs-srfi"
+      :files ("srfi.el" "srfi-data.el"))
```

Comment:
`:files (:default (:exclude "srfi-data-convert.el"))` is better.
Manage with whitelist, you need to modify recipe if you split package file.
Manage with blacklist is you don't need to modify the recipe.